libs/product/state/src/reducers/product/product.reducer.spec.ts

```diff
@@ -37,6 +38,24 @@ describe('Product | Product Reducer', () => {
     });
   });
 
+  describe('when ProductLoadByUrlAction is triggered', () => {
+    let result;
+
+    beforeEach(() => {
+      const productLoadAction = new DaffProductPageLoadByUrl(product.url);
+
+      result = daffProductReducer(initialState, productLoadAction);
+    });
+
+    it('sets loading state to true', () => {
+      expect(result.loading).toEqual(true);
+    });
+
+    it('sets selectedProductId to the product url', () => {
```

Comment:
remove this test, since it is an accidental byproduct of deprecated code, and not a feature we are intending.

Reply:
removed